pandas/tseries/tests/test_timezones.py

```diff
@@ -898,6 +898,46 @@ def test_utc_with_system_utc(self):
         # check that the time hasn't changed.
         self.assertEqual(ts, ts.tz_convert(dateutil.tz.tzutc()))
 
+    def test_tslib_tz_convert_hour_overflow_dst_bug(self):
```

Comment:
can you also put in tests for Timestamp (which are good, but for consistency). you can put in another test function.